Dockerfile

```diff
@@ -18,13 +18,13 @@ USER 10000:10000
 ENTRYPOINT ["/moco-controller"]
 
 # For MySQL binaries
-FROM quay.io/cybozu/mysql:8.0.25.1 as mysql
+FROM quay.io/cybozu/mysql:8.0.26.1 as mysql
```

Comment:
```suggestion
FROM quay.io/cybozu/mysql:8.0.26.2 as mysql
```
?

Reply:
Thank you followup 🙏

> Please add support for 8.0.26 in https://github.com/cybozu-go/moco-agent as well.

I will do it.